loopchain/consensus/runner.py

```diff
@@ -152,26 +158,39 @@ def _generate_genesis_tx(self, tx_versioner):
 
     async def _on_event_broadcast_data(self, event: BroadcastDataEvent):
         if self._block_manager.blockchain.try_update_last_unconfirmed_block(event.data):
+            self._loop.create_task(self._repeat_broadcast_block(event.data))
+
+    async def _repeat_broadcast_block(self, block: "Block"):
+        self._is_broadcasting = True
+        while self._is_broadcasting:
             target_reps_hash = ChannelProperty().crep_root_hash  # FIXME
             self._block_manager.send_unconfirmed_block(
-                block_=event.data,
+                block_=block,
                 target_reps_hash=target_reps_hash,
-                round_=event.data.round_num
+                round_=block.round_num
             )
+            await asyncio.sleep(conf.INTERVAL_BLOCKGENERATION)
```

Comment:
How about defining its own INTERVAL (maybe shorter than INTERVAL_BLOCKGENERATION). Or calculate from it (e.g. INTERVAL_BLOCKGENERATION/2)

Reply:
we need another definition?

Reply:
You mean.. it is block **broadcast** interval, not generation interval?

Reply:
> Discussed in another channel. 
> Nothing is changed. Close this review.

Reply:
> Changed to `conf.INTERVAL_BLOCKGENERATION/2` :rofl: